<ui version="4.0" >
 <class>FilterBuilder_BASE</class>
 <widget class="QWidget" name="FilterBuilder_BASE" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>110</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Filter target:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_FilterTarget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_DisableFilter" >
       <property name="toolTip" >
        <string>Remove all effective filters.</string>
       </property>
       <property name="text" >
        <string>Disable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ApplyFilter" >
       <property name="text" >
        <string>Apply</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_FilterBehaviour" >
       <property name="toolTip" >
        <string/>
       </property>
       <item>
        <property name="text" >
         <string>any</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>all</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>of the following filter rules:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
